Dockerfile.travis

```diff
@@ -1,6 +1,7 @@
 FROM travis/image-inspector-base
 RUN yum install -y \
     git \
+    findutils \
```

Comment:
?

Reply:
@pweil- scripts are using `find` and `xargs`. In the `fedora:26` we don't have them.

Reply:
oh, so this was caused by the other bump.  Got it.

Reply:
@pweil- Fixed